src/util/fetch.spec.js

```diff
@@ -28,3 +28,44 @@ describe('queryParameters', () => {
         );
     });
 });
+
+describe('flattenObject', () => {
+    context('with null', () => {
```

Comment:
please merge the `context` and the `it`, I think it makes tests easier to read when there is only one `it()` test. Besides, that's what is done in the rest of the project.